### DISCUSSION TITLE
Facing issue while downloading through wget

### DISCUSSION THREAD

**Discussion by anjultripathi** [stanfordnlp/CoreNLP#1366] [General]
Hi Team,

We are facing issues in below command :-
wget http://nlp.stanford.edu/software/stanford-corenlp-4.5.2.zip

error:-
ERROR: cannot verify downloads.cs.stanford.edu's certificate, issued by ‘CN=InCommon RSA Server CA,OU=InCommon,O=Internet2,L=Ann Arbor,ST=MI,C=US’:
  Issued certificate has expired.
To connect to downloads.cs.stanford.edu insecurely, use `--no-check-certificate'.

We can't use `--no-check-certificate' as it will badly impact our deployment BDs and container images.

 Can this issue be resolved on the server end?

**Comment by AngledLuffa** [comment]
It works for me now.  Would you try again?
